### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by thebergamo**
image

### CODE REVIEW

**Comment by thebergamo** [comment]
@aledpardo can you take a look :D

File: src/app/Models/CourseModel.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Course } from "../interfaces/course";
+
+export default class CourseModel implements Course {
+  id = ''
```
Comment (aledpardo) [inline src/app/Models/CourseModel.ts]: Consider to define the properties as `public` and add the `;` at end of line.

File: src/app/header/header.component.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #106CC8;
```
Comment (aledpardo) [inline src/app/header/header.component.css]: Just for curiosity, try to use the new CSS variables to handle colors etc.

Comment (aledpardo) [reply src/app/header/header.component.css]: I mean, in future hw

**Review by aledpardo — CHANGES_REQUESTED** [review]
@thebergamo, please do change the classes properties assessor. All the rest seems ok.

File: src/app/Models/CourseModel.ts (hunk 1 above)
Comment (thebergamo) [reply src/app/Models/CourseModel.ts]: done o/

File: src/app/header/header.component.css (hunk 2 above)
Comment (thebergamo) [reply src/app/header/header.component.css]: done o/.

**Review by aledpardo — APPROVED** [review]

**Pull request merged**